docs/sphinx/install.rst

```diff
@@ -263,6 +268,18 @@ executing the commands
 
 .. _local-development-with-vscode:
 
+To develop C++ code, you most likely also want to install the
+`C++ standard library <https://en.cppreference.com/w/cpp/standard_library>`__.
+There is no specific version required to be compatible with CUDA Quantum, but we
+recommend choosing a library that supports C++20 or newer. The C++ standard library
+is almost certainly available via the package manager for your system. Search for a 
+package that contains `libstdc++` in the name, and choose a package with `dev` or 
+`devel`, if available. If you don't find such a package, you can alternatively install
+the complete GCC toolchain to obtain the standard library. If the package you installed
+includes a script to enable that version, you may need to run it to make the library 
+discoverable. You can search for such a script with the command 
+`find / -path '*gcc*' -name enable`.
```

Comment:
Just wanted to double check this command...I tried it on my host system and my Docker container, and it came up with nothing.

Reply:
Well, I don't assume you have multiple gcc versions installed side-by-side, have you? 
The dnf packages make that easy and hence each include such an enable script (which need to be run to use the toolchain) rather than installing it in the default locations.

Reply:
> Well, I don't assume you have multiple gcc versions installed side-by-side, have you? The dnf packages make that easy and hence each include such an enable script (which need to be run to use the toolchain) rather than installing it in the default locations.

That's correct. I do not have multiple versions. I was not familiar with such an `enable` script, so I just wanted to double check. It sounds like it is as you intended.